cmd/cvc-operator/controller/cvc_controller.go

```diff
@@ -271,6 +271,7 @@ func (c *CVCController) createVolumeOperation(cvc *apis.CStorVolumeClaim) (*apis
 	// update the cstorvolume reference, phase as "Bound" and desired
 	// capacity
 	cvc.Spec.CStorVolumeRef = volumeRef
+	cvc.Spec.Policy = volumePolicy.Spec
```

Comment:
this need to be moved up after getting policy..

Reply:
Some changes are not part of this PR, as they are coming with reconcile changes and default policy changes